### ISSUE TITLE
ipxe_nic_drv throughput drops to 10Mbit/s after restart

### ISSUE THREAD

**Issue by jschlatow** [genodelabs/genode#4538]
This can be reproduced on the x260.

According to the ipxe code, this is a known issue for certain intel controllers.The documented workaround is to skip the PHY reset for the broken models. Unfortunately, as the platform driver powers down the pci device when the nic_drv disappears, the workaround becomes ineffective.

**Comment by trimpim** [comment]
I have noticed this also. On Friday I tried to fix the issue (in the ipxr driver only) but wasn't successful.

With your explanation now its clear to we, why I couldn't succeed that way (I forgot to take the `platform_drv` in to account).

On the `imx8` platform driver one could add an other device that also uses the `PHY` clock and as long as one of the drivers still is running, the clock won't be turned off. I'm not sure if this is also possible with the `platform_drv` for the PC platform.

I will try this and write back with my findings.
